packages/synthetic-chain/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@agoric/synthetic-chain",
-  "version": "0.0.7",
+  "version": "0.0.8",
```

Comment:
we use Lerna to publish, which bumps the version. please leave out this commit. after the PR is green  I can cut and publish a release, pushing its commits

Reply:
Awesome, if you could do a release on current `main` that'd be great. I was hoping to bring in the changes in https://github.com/Agoric/agoric-3-proposals/blob/main/packages/synthetic-chain/src/lib/constants.js and remove the reliance on https://github.com/Agoric/agoric-3-proposals/blob/main/proposals/16%3Aupgrade-8/agoric-synthetic-chain-0.0.7-alpha.tgz

Reply:
sorry, I forgot this repo works differently. we do manually bump and then simply `npm publish`. so if you want this to be 0.0.8 this is the right approach. it just won't be in NPM until someone with publish perms runs `npm publish`.

Reply:
published 604071875c0c398cdc4e4b59fbfc6a80a5e08998 as 0.0.8